app/frontend/stores/permit-application-store.ts

```diff
@@ -138,11 +151,11 @@ export const PermitApplicationStoreModel = types
     addPermitApplication(permitapplication: IPermitApplication) {
       self.permitApplicationMap.put(permitapplication)
     },
-    setStatusFilter(status: TFilterableStatus) {
-      if (!status) return
-
-      setQueryParam("status", status)
-      self.statusFilter = status
+    setStatusFilter(statuses: TFilterableStatus[]) {
+      if (!statuses) return
```

Comment:
update type to be statuses: TFilterableStatus[] | undefined to keep it consistent with this conditional